mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -989,41 +996,86 @@
             return false
         }
 
+        override fun onColorSelected(selectedColor: Int) {
+            Log.e(TAG, "onColorSelected($selectedColor)")
+            handleChange(selectedColor, 0)
+        }
+
+        override fun onColorChanged(selectedColor: Int) {
+            Log.e(TAG, "onColorChanged($selectedColor)")
+            handleChange(selectedColor)
+        }
+
+        override fun onValueChange(slider: Slider, value: Float, fromUser: Boolean) {
+            if (fromUser) {
+                handleChange()
+            }
+        }
+
+        override fun onStartTrackingTouch(slider: Slider) {
+            // no-op
+        }
+
+        override fun onStopTrackingTouch(slider: Slider) {
+            handleChange(delay = 0)
+        }
+
+        private fun handleChange(newColor: Int = 0, delay: Long = 1000) {
```

Comment:
1000 seems a bit excessive. Why not keep the previous value of 100?

Reply:
I had some issues with the item updates sending too often. I added a commit where the previous item update is now canceled, if a new updates is queued.